pages/add/form/basic-info-fields.js

```diff
@@ -8,14 +13,10 @@ module.exports = {
     labelClassname: `required`,
     fieldClassname: `form-control`,
     validator: [
-      validator.emptyValueValidator(),
-      validator.maxLengthValidator(80)
```

Comment:
the charLimit does not do auto-validation right now, although I suppose it would make sense for it to do so. We cannot remove the validator functions here (yet).

Reply:
I've filed https://github.com/Pomax/react-formbuilder/issues/54 to address that but for now let's keep these validator functions so that we can be reasonably sure that all the validation we need actually kicks in.